libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -66,6 +66,10 @@ export class CheckboxComponent implements ControlValueAccessor {
     @Input()
     tristateSelectable: boolean = true;
 
+    /** Whenever the checkbox is used inside list, it applies another class to label */
+    @Input()
+    checkboxInList: boolean = false;
```

Comment:
Instead of using an input, can we just use `ContentChildr(CheckboxComponent)` from ListItemDirective and automatically add the class from there?

Reply:
Indeed it would be better

Reply:
Hi @mikerodonnell89  Actually, it's better solution, checkbox doesn't need to know if it's used inside list.